### ISSUE TITLE
Feedback Project 2

### ISSUE THREAD

**Issue by hangyen** [lilbabogina/ga-projects#2]
Hey Gina! Nice job on the project. I like how you spent time summarizing your results and explaining in layman terms what you found. 

My one piece of feedback is on the violin plot -- violins plots are pretty but to most audiences, it's actually very foreign and they have to spend time wrapping their heads around interpreting it. The reason we use plots is so that we can communicate a takeaway clearly and easily and I'm not sure if violin plots accomplish that. Could you have replaced this plot with a bar graph or a histogram and sent the same message? Just something to think about when you're deciding on visuals.

**Comment by lilbabogina** [comment]
Ahh okay! thanks!
